services/asg.py

```diff
@@ -2,16 +2,40 @@
 import boto3
 
 
-# client = boto3.client("")
+client = boto3.client("autoscaling")
 
 
 def autoscaling_group_elb_healthcheck_required():
+    compliant_resources = []
+    non_compliant_resources = []
+    asgs = client.describe_auto_scaling_groups()["AutoScalingGroups"]
+
+    for asg in asgs:
+        if asg.get("HealthCheckType", {}) == "ELB":
+            compliant_resources.append(asg["AutoScalingGroupARN"])
+        else:
+            non_compliant_resources.append(asg["AutoScalingGroupARN"])
```

Comment:
ELB health check는 ASG가 ALB/NLB에 attach 되어 있는 경우에만 필요합니다.
```suggestion
    for asg in asgs:
        if asg["LoadBalancerNames"] or asg["TargetGroupARNs"] and asg["HealthCheckType"] != "ELB":
            non_compliant_resources.append(asg["AutoScalingGroupARN"])
        else:
            compliant_resources.append(asg["AutoScalingGroupARN"])
```